nodejs/eks/nodegroup.ts

```diff
@@ -494,10 +546,12 @@ ${customUserData}
         securityGroups: [nodeSecurityGroupId, ...extraNodeSecurityGroupIds],
         spotPrice: args.spotPrice,
         rootBlockDevice: {
-            encrypted: args.encryptRootBlockDevice || args.encryptRootBockDevice,
-            volumeSize: args.nodeRootVolumeSize || 20, // GiB
-            volumeType: "gp2", // default is "standard"
-            deleteOnTermination: true,
+            encrypted: (args.encryptRootBlockDevice ?? args.nodeRootVolumeEncrypted) ?? false,
```

Comment:
It looks like we had a bug on this line since the `||` operator would have defaulted `false` values to the value of `encryptRootBockDevice`. Let's fix that bug here, but still handle the deprecated `encryptRootBockDevice` parameter. I'd like to remove that in a separate PR.

```suggestion
            encrypted: ((args.encryptRootBlockDevice ?? args.encryptRootBockDevice) ?? args.nodeRootVolumeEncrypted) ?? false,
```